pymongo/monitoring.py

```diff
@@ -1090,8 +1108,8 @@ class ConnectionCheckOutFailedEvent(_ConnectionEvent):
 
     __slots__ = ("__reason",)
 
-    def __init__(self, address: _Address, reason: str) -> None:
-        super().__init__(address)
+    def __init__(self, address: _Address, reason: str, duration: float) -> None:
+        super().__init__(address, 0, duration)
```

Comment:
Using positional arguments here might not be the best idea. Not only can it be hard to follow, but issues with method resolution order (MRO) but creep in.

Reply:
You're suggesting modifying the constructor to keyword args?

Reply:
I'm suggesting calling super using keywords. Then you can mess up the order. My intuition is that this will solve potential MRO issues and if not, at least make clear your intentions if someone needs to fix them in the future.
```
class Baz(Bar):
    def __init__(self, foo, bar, baz):
        super().__init__(bar=bar, foo=foo)
        self.baz = baz
```